src/legacy/core_plugins/kbn_doc_views/public/views/table/table_helper.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { unescape } from 'lodash';
+
+/**
+ * convert <mark> markup of the given text to react
+ * @param text
+ */
+export function replaceMarkWithReactDom(text: string): React.ReactNode {
+  return (
+    <>
+      {text.split('<mark>').map((markedText, idx) => {
+        const sub = markedText.split('</mark>');
+        return (
+          <span key={idx}>
+            <mark>{sub[0]}</mark>
+            {sub[1]}
+          </span>
+        );
+      })}
+    </>
+  );
+}
+// current html of the formatter is angular flavored, this current workaround
+// should be removed when all consumers of the formatHit function are react based
+export function convertAngularHtml(html: string): string | React.ReactNode {
+  if (typeof html === 'string') {
+    const cleaned = html.replace('<span ng-non-bindable>', '').replace('</span>', '');
```

Comment:
This will only replace the first occurrence of the `ng-non-bindable` span, is this intended? Just making sure

Reply:
yes, the formatted value is wrapped by a single span for angular usage. so this should be valid. of course I'll be happy to be able to remove it soon.